assets/tests/consolidated_1_1_tests.yaml

```diff
@@ -651,6 +724,26 @@ tests:
               relation: viewer
             expectation:
               - document:1
+        listUsersAssertions:
+          - request:
+              filters:
+                - user
+              object: document:1
+              relation: viewer
+            expectation:
+              - user:aardvark
+          - request:
+              filters:
+                - group
+              object: document:1
+              relation: viewer
+            expectation:
+          - request:
```

Comment:
We should consider adding one more request assertion just to ensure we don't expand any of our parent ancestors.

e.g.
```yaml
- request:
     filters:
       - group
     object: document:1
     relation: viewer
  expectation:
```

Reply:
🤔 how is that different from lines 735-740?